### DISCUSSION TITLE
Display dialogs anywhere

### DISCUSSION THREAD

**Discussion by gentledepp** [AvaloniaInside/Shell#63] [Q&A]
Hello!
We have some dialogs that we need to show anywhere.
Like: There is no page/route that they actually belong to.

Like some "confirmation dialog" that gives back a result or a "progress dialog"
It would be nice if we could use the shell dialog system for any of our needs. 
But adding those dialogs to each and every page is not an option.

Maybe it would be cool if we could define a base route as a modal? (just guessing)

If that is not the best solution and we should resort to the default dialog system of avalonia, is there an easy way to have the same look and feel (and e.g. entry animation)?

Any suggestions are greatly appreciated!

**Comment by OmidID** [comment]
Hello @gentledepp 

No problem with that. You can call any route from any other route. there is no limitation of heretically. 

But to be more generic, add your page to the root of Shell.
